src/__tests__/__snapshots__/ActionList.tsx.snap

```diff
@@ -11,6 +11,12 @@ exports[`ActionList renders consistently 1`] = `
   line-height: 20px;
 }
 
+.c0[data-has-active-descendant],
+.c0:focus-within {
+  --item-hover-bg-override: none;
+  --item-hover-divider-border-color-override: hsla(214,13%,93%,1);
```

Comment:
I was surprised this is `hsla`, but it seems correct: [`List.tsx#L109`](https://github.com/primer/components/blob/cdc79e633d42f8eaeeec793046423cd09764912d/src/ActionList/List.tsx#L109) sets this variable to `colors.selectMenu.borderSecondary`, which `@primer/primitives` defines in [`light.tsx#L535`](https://github.com/primer/primitives/blob/7a436d367e5c6316a165787e7b89b789245780ac/data/colors/light.ts#L535) to be `lighten(get('scale.gray.2'), 0.03)`. According to the [SCSS docs for `lighten`](https://sass-lang.com/documentation/modules/color#lighten), that function “Increases the HSL lightness of `$color`…”.